### PR TITLE
Support new properties pattern for uPortal 5

### PR DESCRIPTION
**Pull request by kdjohnson**
Support usage of properties files in `portal.home` for widgets.

### CODE REVIEW

File: src/main/webapp/WEB-INF/context/applicationContext.xml [hunk 1]
```diff
@@ -30,48 +30,41 @@
...
+                 | global.properties automatically.  The second file (widgets.properties) is
+                 | (should be) sourced by this module exclusively.
+                 +-->
+                <value>file:${portal.home}/global.properties</value>
+                <value>file:${portal.home}/widgets.properties</value>
+            </list>
+        </property>
     </bean>
 
-    <bean id="propertyResolver" factory-bean="primaryPropertyPlaceholderConfigurer" factory-method="getPropertyResolver"/>
```
Comment (drewwills) [inline src/main/webapp/WEB-INF/context/applicationContext.xml]: Is this item no longer needed?  Do you know anything about it @jhelmer-unicon?

**Comment by drewwills** [comment]
Looks pretty good.

I'm concerned there may be portlet and/or servlet sub-contexts that will need to "pull in" the `PropertySourcesPlaceholderConfigurer` in a manner something like this:

  - https://github.com/Jasig/CalendarPortlet/blob/master/src/main/resources/context/portlet/calendar.xml#L36

**Comment by ChristianMurphy** [comment]
Hey @kdjohnson! :wave: 
Have you had a chance to take a look at pulling in `PropertySourcesPlaceholderConfigurer` in serverlet sub-contexts?

**Comment by kdjohnson** [comment]
@ChristianMurphy I have not had a chance yet. I'll likely have some time to look at it next week some time.

**Comment by kdjohnson** [comment]
@ChristianMurphy I've re-added the bean.

**Comment by ChristianMurphy** [comment]
[Invoking Lazy Consensus](https://www.apache.org/foundation/voting.html#LazyConsensus).
Unless there are any objections, this will be merged later today.

**Comment by drewwills** [comment]
@ChristianMurphy & @kdjohnson -- As it stands, none of the subcontexts have access to properties defined in the `propertySourcesPlaceholderConfigurer`.

Does that matter?  TBH I'm not positive;  but it seems to me that not including support in the subcontexts could easily trip someone up in the future, and that probably it's a good idea for us to take care of it.

We would need something like this...

  - https://github.com/Jasig/CalendarPortlet/blob/master/src/main/resources/context/portlet/calendar.xml#L36

in each of the files here...

  - https://github.com/Jasig/JasigWidgetPortlets/tree/master/src/main/webapp/WEB-INF/context/portlet

This is the same feedback I gave on Oct 4.

**Comment by kdjohnson** [comment]
Added bean inheritance to portlet contexts.

**Comment by drewwills** [comment]
LGTM :+1:

**Pull request merged**